addons/captives/functions/fnc_doApplyHandcuffs.sqf

```diff
@@ -21,6 +21,6 @@ params ["_unit", "_target"];
 
 playSound3D [QUOTE(PATHTO_R(sounds\cable_tie_zipping.ogg)), objNull, false, (getPosASL _target), 1, 1, 10];
 
-[QGVAR(setHandcuffed), [_target, true], [_target]] call CBA_fnc_targetEvent;
+[QGVAR(setHandcuffed), [_target, true, _caller], [_target]] call CBA_fnc_targetEvent;
```

Comment:
`_caller` isn't defined, I think this should just be `_unit`
```suggestion
[QGVAR(setHandcuffed), [_target, true, _unit], [_target]] call CBA_fnc_targetEvent;
```

Reply:
or a better fix might be to change all other refs of `_unit` to `_caller` to keep consistancy with setHandcuffed